### ISSUE TITLE
Improving the Chunk-Filter (even more)

### ISSUE THREAD

**Issue by chland** [Querz/mcaselector#3]
I don't know if you want to add more features to the tool but after testing it for a few minutes i noticed two small things that could improve the chunk-filter (which, by the way, is an incredible feature).

1. a "Select" option -> right now you can only Export/Delete the result of the filter - it would be nice if there was a third option, "Select", which would just select all the chunks in the main window  that match the filter - that way you could "preview" what chunks are affected before actually deleting/exporting them
2. when you Delete/Export chunks using the filter it would be nice if the tool would display a confirmation-dialog with a message like "You are about to delete/export 12345 chunks. Do you really want to do this?" and if the filter matches a whole world: "This will delete/export your whole world. Do you really want to do this?"

**Comment by Querz** [comment]
I implemented the "Select" option for the chunk filter and added confirmation dialogs to all delete and export actions.
I still have some features though i want to add before i create another release, like having the chunk filter and the NBT changer only apply to a selection.
If you want to try the "Select" option, you can either build it yourself from the master branch or use this one: 
[mcaselector-1.4-snapshot.jar.zip](https://github.com/Querz/mcaselector/files/2404350/mcaselector-1.4-snapshot.jar.zip)

**Comment by chland** [comment]
I did a few quick tests and the "Select"-option seems to work perfectly :) Its pretty interesting to use it in combination with the InhabitedTime condition.

And of course you're right about the problems with displaying the actual number of chunks a filter would delete/export. I only used the filter on my new single-player world which is still kinda small but i guess filtering through an old world with half a million chunks or something like that could take quite some time.

Thanks for adding the new option so quickly!
